lib/index.js

```diff
@@ -79,19 +79,21 @@ internals.registrationConfig = (options) => {
     return config;
 };
 
-internals.initialize = function (server, next) {
+internals.initialize = (server, next) => {
 
-    const rootCollector = internals.state(server.root).collector;
+    const collector = internals.state(server.root).collector;
+    const rootKnex = Hoek.reach(collector, 'knexGroup.knex') || null;
```

Comment:
Ahh I see!  Yes, this seems like a bug.  I believe that's actually one of the untested lines, which is why the tests pass.  Will fix.  Nice find!